test/spec.js

```diff
@@ -128,7 +128,7 @@ test('attempt timeout causes retry exhaustion', async t => {
 });
 
 test('operation timeout before attempt timeout', async t => {
-    const retry = dread({ attempts: 2, timeout: 10 });
+	const retry = dread({ attempts: 2, timeout: 10 });
```

Comment:
mixed tabs/spaces  (we're missing an .editorconfig)

Reply:
Should I add one?

Reply:
Sure, grab one from https://github.com/dougmoscrop/serverless-http